service/metadata.yaml

```diff
@@ -71,3 +71,89 @@ telemetry:
       gauge:
         async: true
         value_type: int
+
+    receiver.produced.items:
```

Comment:
Does this also need a consumed items?

Reply:
See https://github.com/open-telemetry/opentelemetry-collector/pull/12812#discussion_r2039885561